README.md

```diff
@@ -238,6 +238,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 * [OUISRC/muro](https://github.com/OUISRC/muro) — A new Forum use rust, rocket, diesel, postgresql, and responsive design [<img src="https://api.travis-ci.org/OUISRC/muro.svg?branch=master">](https://travis-ci.org/OUISRC/muro)
 
+
```

Comment:
This adds (back?) an extra newline. Should I rebase and force push the branch?

Reply:
I don't think it matters too much, but you are welcome to do so

Reply:
Done. :+1: